content/themes/chappress/main.scss

```diff
@@ -62,7 +62,7 @@ a {
   line-height: 20px;
 }
 
-.custom-logo-link {
-  background-image: url("http://localhost:8222/wp-content/themes/chappress/assets/images/cu-logo.png");
+.chapman-logo-link {
+  background-image: url("https://chappress-staging.chapman.edu/wp-content/themes/chappress/assets/images/cu-logo.png");
```

Comment:
A relative url works here, yes?

    background-image: url("/wp-content/themes/chappress/assets/images/cu-logo.png");

Reply:
Removed that css class, it was left accidentally and wasn't loading.